questions/accessibility-aria.md

```diff
@@ -0,0 +1,36 @@
+### What is ARIA and when should you use it?
+
+#### Answer
+
+ARIA stands for "Accessible Rich Internet Applications". ARIA code is HTML. It provides additional attributes in the development of web applications to offer screen reader users more context about dynamic components, including the component's role, name, and state. It should only be used when an HTML element equivalent is not available or lacks full support. ARIA is semantic, and doesn't actually make your component dynamic.
+
+An example using ARIA:
+
+```
+<div 
+  role="combobox"
+  aria-expanded="false"
+  aria-owns="ex1-grid"
+  aria-haspopup="grid"
+  id="ex1-combobox">
+  ...
```

Comment:
You can remove these `...` I guess. 😃

Reply:
No problem. An indicator that more code should go there, but prob not the best way to express that idea.

Reply:
Oh my wrong .... Sorry, I'll fix this.